content-security-policy/embedded-enforcement/support/echo-allow-csp-from.py

```diff
@@ -1,10 +1,12 @@
+from wptserve.utils import isomorphic_decode
+
 import json
 def main(request, response):
-    headers = [("Content-Type", "text/html")]
-    if "allow_csp_from" in request.GET:
-        headers.append(("Allow-CSP-From", request.GET["allow_csp_from"]))
-    message = request.GET["id"]
-    return headers, '''
+    headers = [(b"Content-Type", b"text/html")]
+    if b"allow_csp_from" in request.GET:
+        headers.append((b"Allow-CSP-From", request.GET[b"allow_csp_from"]))
+    message = request.GET[b"id"]
+    return headers, u'''
```

Comment:
Here, can we instead use `b'''` and avoid `isomorphic_decode` for the inputs? That should be less code and doing less conversion overall, even if the result is the same.

This is my first review like this, so if you've gotten the opposite suggestion from other reviewers, please let me know :)

Reply:
I guess either type is accepted. I will change this to bytes.